Makefile

```diff
@@ -90,10 +90,24 @@ all: image
 
 build:
 	@mkdir -p bin
-	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/...
+	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/nfd-gc
+	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/nfd-worker
+	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/nfd-master
+	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/nfd-topology-updater
```

Comment:
Maybe you could use a variable to simplify and re-use also in `install:`target below

```
BUILD_BINARIES := nfd-master nfd-worker nfd-topology-updater nfd-gc

build:
    $(GO_CMD) build -v -o bin $(BUILD_FLAGS) $(foreach bin, $(BUILD_BINARIES), ./cmd/$(bin))
```